packages/@webex/plugin-meetings/test/integration/spec/converged-space-meetings.js

```diff
@@ -0,0 +1,126 @@
+import { config } from 'dotenv';
+
+import {assert} from '@webex/test-helper-chai';
+import {skipInNode} from '@webex/test-helper-mocha';
+
+import testUtils from '../../utils/testUtils';
+import webexTestUsers from '../../utils/webex-test-users';
+
+config();
+
+skipInNode(describe)('plugin-meetings', () => {
+  describe('converged-space-meeting', () => {
+    let users, alice, bob, chris;
+    let meeting = null;
+    let space = null;
+
+    before(() => webexTestUsers.generateConvergedTestUsers({
+        count: 3,
+        whistler: process.env.WHISTLER || process.env.JENKINS,
+      }).then((userSet) => {
+        users = userSet;
+        alice = users[0];
+        bob = users[1];
+        chris = users[2];
+        alice.name = 'alice';
+        bob.name = 'bob';
+        chris.name = 'chris';
+      }).then(() => Promise.all([
+        testUtils.syncAndEndMeeting(alice),
+        testUtils.syncAndEndMeeting(bob),
+      ])).catch((error) => {
+        console.log(error);
+      })
+    );
+
+    it('user "alice" starts a space', () =>
+      alice.webex.internal.conversation.create({
+        participants: [bob, chris],
+      }).then((conversation) => {
+        assert.lengthOf(conversation.participants.items, 3);
+        assert.lengthOf(conversation.activities.items, 1);
+        space = conversation;
+      }).then(() => Promise.all([
+        alice.webex.meetings.meetingInfo.fetchMeetingInfo(space.url, 'CONVERSATION_URL'),
+        alice.webex.meetings.meetingInfo.fetchMeetingInfo(space.url),
+      ])).then(([destinationWithType, destinationWithoutType]) => {
+        assert.exists(destinationWithoutType);
+        assert.exists(destinationWithType);
+        assert.exists(destinationWithoutType.body.meetingNumber);
+        assert.exists(destinationWithType.body.meetingNumber);
+      }),
+    );
+
+    it('user "alice" starts a meeting', () =>
```

Comment:
I've created a local serial workflow that bails/aborts whenever a single test in a chain fails. This will be elevated later. But should work here for now.